library/ssl_tls12_client.c

```diff
@@ -1732,6 +1732,82 @@ static int ssl_parse_server_psk_hint(mbedtls_ssl_context *ssl,
 }
 #endif /* MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED */
 
+#if defined(MBEDTLS_KEY_EXCHANGE_RSA_ENABLED)
```

Comment:
This addition does not look correct and was not present in the previously-approved version of this PR. Something must have gone wrong when amending the last commit.